install_files/securedrop-ossec-server/DEBIAN/control

```diff
@@ -6,7 +6,7 @@ Homepage: https://securedrop.org
 Package: securedrop-ossec-server
 Version: 2.8.2+0.5
 Architecture: amd64
-Depends: ossec-server,securedrop-keyring
+Depends: ossec-server,securedrop-keyring,securedrop-config
```

Comment:
soooo as a totally side discussion in my PR for #2592 - i made it a point not to install tor on ossec if one enables tor over local network. I don't know if that was the right call (im on the fence) - I'm just making a note that here you are making this package a pre-requisite for ossec server where in my PR I assume tor might not be installed on the mon server... Once again, totally outside the scope of this ticket but I would like to convo about what should be the right call there.

Reply:
Hm, interesting. To be clear, the `securedrop-config` package does not actually install tor, it just configures the FPF apt mirror _for_ tor. So there shouldn't be any conflict there.

But the discrepancy you point out is the direct result of trying to support customizations of deployments, so we haven't seen the last of that (related: #1129).

Reply:
> Hm, interesting. To be clear, the securedrop-config package does not actually install tor, it just configures the FPF apt mirror for tor. So there shouldn't be any conflict there.

Yeah thats a very good point but just something I figured I'd point out since it caught my attention. 

Regarding #1129 though-- i really dont want administrators to make those kinds of unique changes. If they are doing that, we need to try to officially support the features they are trying to add.